.env.ci

```diff
@@ -17,6 +17,7 @@ NATS_ENVIRONMENT_NAME=$NATS_ENVIRONMENT_NAME
 
 # ENS
 CHAIN_ID=73799
+CHAIN_NAME=volta
```

Comment:
I think the chain info is relevant to more than just ENS (i.e. it's obviously relevant to DID caching as well), so I think I would split it out into another section
```suggestion
# CHAIN CONNECTION
CHAIN_ID=73799
CHAIN_NAME=volta

# ENS
```